docs/guide/integrations/multistore.md

```diff
@@ -204,3 +244,119 @@ or
   "
 ></router-link>
 ```
+
+
+## Useful _Helpers_
+
+### How to get current store view?
+
+Here is a helper method to get the value of current store view.
+
+You just need to import `currentStoreView` function from `core/lib/multistore` as example follows : 
```

Comment:
`core/lib/multistore` -> `@vue-storefront/core/lib/multistore`
Someone will copy paste short one - one day and will have problem.

Reply:
Right